docs/content/release_notes/changelog_1.7.0.md

```diff
@@ -1,6 +1,10 @@
 # 1.7.0 (September 2021)
 
-IBM® block storage CSI driver 1.7.0 added new support and enhancements:
+IBM® block storage CSI driver 1.7.0 added new support and enhancements and resolved the following issue:
 - Now supports the CSI Topology feature
```

Comment:
isn't it weird to end the sentence with "the following issue" and then start with the new support and enhancements?

Reply:
updated wording in all three prs

Reply:
I don't see the change in this PR specifically

Reply:
please check again. i had forgotten to push my cange earlier.